cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -135,13 +140,9 @@ <h3 class="h4">
    ========================================================================== #}
 
 {% macro _text(text) %}
-
-<p class="u-mb0">
```

Comment:
That seems strange. I'm not sure why the text entered in wagtail wouldn't be inside the `p.u-mb0`. Did you look at the actual source, or just look at it in the Inspector? If the latter, I wonder if the browser interprets an invalid nested paragraph as two consecutive paragraphs. Regardless, the change makes sense if no one has cared about the bottom margin not being zeroed-out to this point.